### ISSUE TITLE
Suggestion to adjust color scheme for sbase GUI in Mac's dark mode.

### ISSUE THREAD

**Issue by coseto6125** [seleniumbase/SeleniumBase#1862]
Hello, I noticed that the current color scheme of the sbase gui is not very user-friendly for Mac's dark mode. 
The white background color of the case title is overlapping with the background and making it difficult to read. 
Additionally, the bright blue font color of the prompt is a little hard on the eyes.

I would like to suggest selecting a more balanced color scheme or fixing the background color to improve the readability in dark mode. 

If considering adjusting colors based on light/dark color schemes, 
you may refer to https://github.com/albertosottile/darkdetect for inspiration.

<img width="1641" alt="image" src="https://user-images.githubusercontent.com/80243681/235377487-71cd9628-3fe2-4b7b-b8bf-705abddb5489.png">

```diff
sb_commander.py
- 294   tk.Label(root, text=run_display, fg="blue").pack() 
+ 294.  tk.Label(root, text=run_display, fg="gray").pack() 
~~
- 309                  bg="white",
+ 309                  bg="gray",
~~
- 335     fg="blue", 
+ 335     fg="gray", 
~~

```

https://github.com/seleniumbase/SeleniumBase/blob/e144cecfbf7eca7d39e7c7ffb0549bff484c9d02/seleniumbase/console_scripts/sb_commander.py?plain=309#L305-L313

https://github.com/seleniumbase/SeleniumBase/blob/e144cecfbf7eca7d39e7c7ffb0549bff484c9d02/seleniumbase/console_scripts/sb_commander.py#L332-L336

https://github.com/seleniumbase/SeleniumBase/blob/e144cecfbf7eca7d39e7c7ffb0549bff484c9d02/seleniumbase/console_scripts/sb_commander.py#L294

**Comment by mdmintz** [comment]
Sounds good. I can get some neutral colors in the next release so that text is visible in both light and dark modes.

**Comment by mdmintz** [comment]
This was resolved in ``4.14.5`` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.14.5
